sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/DigitalTwinsModelFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.DigitalTwins.Core
```

Comment:
I wonder if they'd prefer factories to facilitate mocking should be in a sub namespace? It isn't mentioned in the guidance, but to me it seems misleading to include it in the root namespace.

@KrzysztofCwalina any guidance here?

Reply:
From the guidance I linked above

> Implement a static class called <service>ModelFactory in the same namespace as the model types:

In our case, these model types are in the base namespace, so this factory is in the base namespace as well